rpcs3/rpcs3qt/gui_application.cpp

```diff
@@ -64,6 +64,7 @@ std::unique_ptr<raw_mouse_handler> g_raw_mouse_handler;
 
 gui_application::gui_application(int& argc, char** argv) : QApplication(argc, argv)
 {
+	std::setlocale(LC_ALL, "C"); // On linux Qt changes to system locale while initializing QCoreApplication
```

Comment:
I think it should be LC_NUMERIC to prevent weird localization nonsense.

Reply:
I don't think it matters but since the documentation uses numeric it I guess we should do so too.